=== tests/cases/compiler/redeclareParameterInCatchBlock.ts ===
try {

} catch(e) {
>e : any
>类型别名 :=> 任意

    const e = null;
>e : any
>类型别名 :=> 任意
>null : null
>类型别名 :=> 空值
}

try {

} catch(e) {
>e : any
>类型别名 :=> 任意

    let e;
>e : any
>类型别名 :=> 任意
}

try {

} catch ([a, b]) {
>a : any
>类型别名 :=> 任意
>b : any
>类型别名 :=> 任意

    const [c, b] = [0, 1];
>c : number
>类型别名 :=> 数字
>b : number
>类型别名 :=> 数字
>[0, 1] : [number, number]
>0 : 0
>1 : 1
}

try {

} catch ({ a: x, b: x }) {
>a : any
>类型别名 :=> 任意
>x : any
>类型别名 :=> 任意
>b : any
>类型别名 :=> 任意
>x : any
>类型别名 :=> 任意

}

try {

} catch(e) {
>e : any
>类型别名 :=> 任意

    function test() {
>test : () => void

        let e;
>e : any
>类型别名 :=> 任意
    }
}

